packages/rottweiler/rottweiler.spec

```diff
@@ -0,0 +1,31 @@
+%global _cross_first_party 1
+%undefine _debugsource_packages
+
+Name: %{_cross_os}rottweiler
+Version: 0.1.0
+Release: 1%{?dist}
+Summary: Bottlerocket storage encryption helper
+License: Apache-2.0 OR MIT
+URL: https://github.com/bottlerocket-os/bottlerocket
+
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%prep
+%setup -T -c
+%cargo_prep
+
+%build
+%cargo_build --manifest-path %{_builddir}/sources/Cargo.toml \
+    -p rottweiler
```

Comment:
I assume the `cross_scan_attribution` isn't needed because of this?

https://github.com/bottlerocket-os/bottlerocket-core-kit/blob/cfe9fded8063b79402e517077b11f77596bdcd73/packages/os/os.spec#L750

Reply:
> I assume the `%cross_scan_attribution` isn't needed because of this?

Yes, we'll create the right attribution files based on the scan in `os`.